README.md

```diff
@@ -141,9 +168,10 @@ All grasshopper services and apps can be built as [Docker](https://docs.docker.c
     }
     ```
 
-    **Note:** If using `boot2docker`, the following with get you the  `docker-provided-ip`:
+    Other URLs:
 
-        $ boot2docker ip
+    - UI = http://{{docker-provided-ip}}
+    - Geocoder = http://{{docker-provided-ip}}/geocode/{{address}}
```

Comment:
That is true.  It is actually available at both resources.  `:8080/geocode/` takes you directly to the Scala app, while `/geocoder/geocode/` (port `80`) is available via `grasshopper-ui`'s Nginx setup.  I agree, its probably more appropriate to leave these docs as `8080` for consistency with the other containers.